.azure-pipelines-templates/test.yml

```diff
@@ -27,13 +27,10 @@ steps:
   condition: or(failed(), canceled())
   displayName: 'Upload logs (only on failure)'
 
-- script: 'xsltproc --stringparam suiteName "$(Agent.MachineName) ${{ parameters.suite_name_suffix }}" ../tests/infra/ctest_to_junit.xslt Testing/*/Test.xml | sed -r "s/\[NON-XML-CHAR-0x1B\]\[([0-9]*)m//g"> JUnit.xml'
```

Comment:
I'd be tempted to keep a preprocessing step where we run sed to strip out the non-xml-chars, so we can still read this in the ADO web view? Non-critical though.

Reply:
Yes, I was hoping to find something in ctest, but I think this is the best solution we have.